### ISSUE TITLE
Create a Vagrant environment for development

### ISSUE THREAD

**Issue by paniag** [datajango/specljs#1]

**Comment by datajango** [comment]
a branch?

**Comment by paniag** [comment]
No, the idea was to create a work item for me to complete, only I couldn't
find a way to assign it to myself.  Maybe I'm using GitHub wrong and should
have created the issue in my fork, but that seems a poor design as it
fragments the project-level issues.  I'll try again today.

This but it's mostly complete, by the way.  I just wanted to record and
communicate the work.
